invenio_app_ils/ui/src/invenio_app_ils/common/state/PatronCurrentLoans/__tests__/actions.js

```diff
@@ -64,7 +68,7 @@ describe('Patron current loans tests', () => {
 
       store.dispatch(actions.fetchPatronCurrentLoans(2)).then(() => {
         expect(mockFetchPatronCurrentLoans).toHaveBeenCalledWith(
-          '(patron_pid:2 AND state:ITEM_ON_LOAN)&sort=-mostrecent'
+          '(patron_pid:2 AND state:(ITEM_ON_LOAN))&sort=-mostrecent'
```

Comment:
Why the parentheses here? I think was also correct before no?

Reply:
For current loans we had hardcoded the state in the query, I switched it to the `invenioConfig.circulation.loanActiveStates` which is a list of states, so the form of our query changes, thats why the tests where updated with parentheses.